src/style.css

```diff
@@ -1,3 +1,81 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  padding: 0 50px;
+  height: auto;
+  position: relative;
+  display: flex;
+  align-items: stretch;
+  justify-content: space-between;
+}
+
+.logo {
+  margin: 10px 0;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  text-transform: uppercase;
+  display: flex;
+  align-items: stretch;
+  justify-content: center;
+}
+
+.items {
+  display: flex;
+  align-items: stretch;
+  padding-inline-start: 0;
+  margin: 0;
+}
+
+.item {
+  position: relative;
+  display: flex;
+  align-items: stretch;
+  list-style-type: none;
+  font-weight: 500;
+  font-size: 12px;
+  margin: 0 10px;
+}
+
+.item a {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  cursor: pointer;
+  color: #000;
+}
+
+.item-1 {
+  margin-left: 0;
+}
+
+.item-8 {
+  margin-right: 0;
+}
```

Comment:
better to use :first-child and :last-child
and remove all classes like `item-1`